public/app/plugins/panel/canvas/CanvasPanel.tsx

```diff
@@ -61,11 +62,16 @@ export class CanvasPanel extends Component<Props, State> {
       moveableAction: false,
     };
 
+    // TODO: Will need to update this approach for dashboard scenes
+    // migration (new dashboard edit experience)
+    const dashboard = getDashboardSrv().getCurrent();
+    const allowEditing = this.props.options.inlineEditing && dashboard?.editable;
+
```

Comment:
editable is not the same as isEditing, 

https://github.com/grafana/grafana/blob/main/public/app/features/dashboard-scene/utils/DashboardModelCompatibilityWrapper.ts#L61

Reply:
but you could add a new property isEditing on DashboardModelCompatibilityWrapper if you want

Reply:
In its current form this PR addresses the legacy dashboard case which I think is still worth addressing as dashboard scenes is still in public preview

Created this follow up GitHub issue to address figuring this out for new dashboards based on scenes. I think for this to work as expected I think the dashboard scene will need to emit a state update (like editing is turned off / on) that a panel can listen to and respond to accordingly

https://github.com/grafana/grafana/issues/87948